params.go

```diff
@@ -72,7 +72,7 @@ var (
 	)
 	foragingDeerInputScaler = flag.Float64(
 		"foragingDeerInputScaler",
-		12,
+		15,
```

Comment:
Please changes all these params back to defaults - they won't let me merge these into main repo. @P-A-Ha @justma360